### PR TITLE
Add ALC1220 layout-id=99 for MiBook 2019 and ALC262 for MS-7847 layout-id 66 and ALC269VC for Hasee K790s layout-id 47

### PR DESCRIPTION
**Pull request by Dynamix1997**

### CODE REVIEW

**Comment by Andrey1970AppleLife** [comment]
You have an error here: https://github.com/acidanthera/AppleALC/pull/491/files#diff-fab121b01685c7d90da5f5d47c006bdeR2047
There are no lines:
<key>AFGLowPowerState</key>
<data>AwAAAA==</data>
<key>Codec</key>

File: Resources/PinConfigs.kext/Contents/Info.plist [hunk 1]
```diff
@@ -2044,6 +2044,21 @@
 					<key>LayoutID</key>
 					<integer>28</integer>
 				</dict>
+				<dict>
+					<string>ALC262 for MS-7847</string>
+					<key>CodecID</key>
```
Comment (Dynamix1997) [inline Resources/PinConfigs.kext/Contents/Info.plist]: <key>AFGLowPowerState</key>
 					<data>AwAAAA==</data>

**Comment by Dynamix1997** [comment]
Info.plist Fixed

**Comment by Andrey1970AppleLife** [comment]
Rename file Platforms.xml to Platforms66.xml

**Comment by Andrey1970AppleLife** [comment]
You added the file Platforms.xml in ALC262, but in Info.plist file name Platforms**66**.xml.zlib.
I two times wrote you that you renamed file.
If you ignore, PR will be closed.

**Comment by Andrey1970AppleLife** [comment]
It everything works for you?
I have doubts because you make many mistakes.

**Comment by Dynamix1997** [comment]
Yes, I'm ashamed that my negligence has inconvenienced you. This is my first time using GitHub. I'll be careful next time

**Pull request merged**